### DISCUSSION TITLE
Rotation

### DISCUSSION THREAD

**Discussion by gitalm** [xournalpp/xournalpp#3622] [Q&A] [answered]
Dear Team, Xournal++ is really a wonderful software. 
My version cannot rotate any images. I think it worked some time ago, but now i cannot find any setting to allow this .
Where can I find the setting to allow rotation (Just find snap to certain angels)

And probably there is also a chance to embed images smaller, Images from screenshots and by Dragging are always rendered very big. It is possible to adjust this.

Thanks for the help

![grafik](https://user-images.githubusercontent.com/7422149/143542878-d117d9b2-0b32-4f81-9ca1-8f4df84d235b.png)

**Comment by rolandlo** [comment, marked as answer]
I'm glad you like Xournal++. We don't have image rotation support yet. It has been a long standing feature request, that was connected to a new file format https://github.com/xournalpp/xournalpp/issues/918

About the size of pasted images: It depends on the display dpi settings in the `Zoom` panel of menu `Edit > Preferences`. So try adjusting that.

**Reply by gitalm** [reply]
Thanks for your help, the trick with dpi is working!
